package.json

```diff
@@ -10,10 +10,12 @@
     "bootstrap": "lerna bootstrap",
     "check-updates": "./scripts/check-updates",
     "clean": "rm -rf coverage; lerna exec -- rimraf lib dist",
-    "precommit": "lint-staged && npm run todo-staged",
+    "precommit": "lint-staged",
     "commitmsg": "commitlint -e $GIT_PARAMS",
     "commit": "git-cz",
     "build": "lerna run --parallel build",
+    "generate:todos": "leasot -x --reporter markdown 'packages/*/src/**.js'",
```

Comment:
This is your printing script. It's just dumps them to stdout.

Reply:
Is this ok looking in terminal though? Sorry am on phone so can’t check. I mean since reporter is markdown instead of default table.

Reply:
Ah well yeah. Then may generate and build do not use one another and duplicate the globa little  If not otherise possible.